pkg/policy/k8s/service.go

```diff
@@ -34,7 +35,7 @@ func (p *policyWatcher) isSelectableService(svc *k8s.Service) bool {
 	if svc == nil {
 		return false
 	}
-	return p.config.EnableHighScaleIPcache || svc.IsExternal()
+	return p.config.EnableHighScaleIPcache
```

Comment:
I think this whole function can just be `return svc != nil` -- or even just removed.

Reply:
Updated.